### PR TITLE
removed static issuer t&c content, pulled from the api instead

### PR DESCRIPTION
**Pull request by ssteveli**

### CODE REVIEW

File: pebble-web/src/modules/authentication/controllers/cardActionController.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
 'use strict';
 
-appControllers.controller('CreditCardController', [
-    '$http','$scope', '$rootScope', '$location', 'creditCardService', 'pagareConfig','halService',
-    'deviceService','$routeParams','AccessToken',
-    function ($http, $scope, $rootScope, $location, creditCardService, pagareConfig, halService,
-    deviceService, $routeParams, AccessToken
-) {
+appControllers.controller('CreditCardController', function ($http, $scope, $rootScope, $location, creditCardService) {
```
Comment (bpwalford) [inline pebble-web/src/modules/authentication/controllers/cardActionController.js]: I've run into module injection errors on deployment when using implicit dependency injection, I think it's safest to always use explicit injection.

File: pebble-web/src/modules/authentication/controllers/issuerTermsController.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+appControllers.controller('IssuerTermsController', function ($scope, $routeParams, $http, $sce, $location, creditCardService) {
```
Comment (bpwalford) [inline pebble-web/src/modules/authentication/controllers/issuerTermsController.js]: same as above

File: pebble-web/src/modules/authentication/controllers/issuerTermsController.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    creditCardService.getCreditCard($routeParams.creditCardId)
+        .then(function (card) {
+            $scope.card = card;
+
+            var termsUrl = card.termsAssetReferences[0]._links.self.href || undefined;
+            if (termsUrl) {
+                $http.get(termsUrl).success(function (result) {
```
Comment (bpwalford) [inline pebble-web/src/modules/authentication/controllers/issuerTermsController.js]: Angular $http docs have a deprecation warning for the success and error methods, it suggests to always use the then method instead.

File: pebble-web/src/modules/authentication/controllers/cardActionController.js (hunk 1 above)
Comment (ssteveli) [inline pebble-web/src/modules/authentication/controllers/cardActionController.js]: Ugh, you're right.... I did some reading on the implicit injection and it's our minify tools that cause the problems downstream...  with the explicit the code is just so bulky, will adjust!

File: pebble-web/src/modules/authentication/controllers/issuerTermsController.js (hunk 2 above)
Comment (ssteveli) [inline pebble-web/src/modules/authentication/controllers/issuerTermsController.js]: fixed

File: pebble-web/src/modules/authentication/controllers/issuerTermsController.js (hunk 3 above)
Comment (ssteveli) [inline pebble-web/src/modules/authentication/controllers/issuerTermsController.js]: fixed

**Pull request merged**